react-components/src/components/RevealTopbar/RuleBasedOutputsContainer.tsx

```diff
@@ -0,0 +1,103 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { useState, type ReactElement, useEffect, useMemo } from 'react';
+
+import { Button, Dropdown, Flex, Icon, Menu, Switch } from '@cognite/cogs.js';
+
+import { RuleBasedOutputs } from '../RuleBasedOutputs/RuleBasedOutputs';
+import { RULE_BASED_OUTPUTS_VIEW } from '../RuleBasedOutputs/constants';
+import { FdmSDK } from '../../utilities/FdmSDK';
+import { useSDK } from '../RevealCanvas/SDKProvider';
+import { type RuleAndEnabled } from '../RuleBasedOutputs/types';
+import { useTranslation } from '../i18n/I18n';
+
+export const RuleBasedOutputsContainer = (): ReactElement => {
+  const [ruleInstances, setRuleInstances] = useState<RuleAndEnabled[]>();
+  const [currentRuleSetEnabled, setCurrentRuleSetEnabled] = useState<any>();
+  const sdk = useSDK();
+  const fdmSdk = useMemo(() => new FdmSDK(sdk), [sdk]);
+
+  const { t } = useTranslation();
+
+  useEffect(() => {
+    // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+    const getAllInstances = async () => {
+      const versionedPropertiesKey = `${RULE_BASED_OUTPUTS_VIEW.externalId}/${RULE_BASED_OUTPUTS_VIEW.version}`;
+
+      const filter = {
+        in: {
+          property: [RULE_BASED_OUTPUTS_VIEW.space, versionedPropertiesKey, 'shamefulOutputTypes'],
```

Comment:
We better remind each other to fix this data model and field name at some point :sweat_smile:

Reply:
yeah :D